client/blocks/signup-form/index.jsx

```diff
@@ -418,7 +418,6 @@ class SignupForm extends Component {
 			<div>
 				<FormLabel htmlFor="email">{ this.props.translate( 'Your email address' ) }</FormLabel>
 				<FormTextInput
-					autoFocus={ ! this.props.isSocialSignupEnabled }
```

Comment:
I'm not sure if we want to do this. I understand the reasoning behind removing it, but it's there for a purpose already. Would be nice to get a designer approve this before we 🚢 .

Reply:
Could you provide more context for the reasoning?

Reply:
We usually avoid `autofocus` usage for accessibility reasons, because screen reader users might end up in the middle of a form, without context or information about that form. Instead, it might make sense for them to go through the title/subtitle and the rest of the content that's above the form first. 

Whether this case is one of those - I'd ask a designer.

Reply:
@scruffian I'm obeying the a11y linter to be perfectly honest. Some context here:

https://w3c.github.io/html/sec-forms.html#autofocusing-a-form-control-the-autofocus-attribute

cc: @ryelle for a11y input